src/styles/styles.js

```diff
@@ -511,6 +602,12 @@ const styles = {
 
     noOutline: addOutlineWidth({}, 0),
 
+    labelLabelSupporting: {
```

Comment:
Sorry, my fault in the last review 🤦 
```suggestion
    textLabelSupporting: {
```